3IPs/3ip-0.md

```diff
@@ -0,0 +1,15 @@
+---
+3ip: 0
+title: 3IP Purpose and Guidelines
+status: Active
+author: Joel Thorstensson <oed@3box.io>
+created: 2019-01-16
+---
+
+## What is a 3IP?
+
+3IP stands for 3Box Improvement Proposal. A 3IP document contains information describing new features or other types of changes to the 3Box community. They are intended to be similar to EIPs and BIPs of Ethereum and Bitcoin respectively, but we are starting of with less formalities for now.
+
+## Creating a 3IP
+Simply copy [3ip-X.md](./3ip-X.md) and start writing. Once you have a proposal ready make a PR to this repo adding your file. The number of your PR becomes the number of the 3ip.
```

Comment:
Change to: "... Once your proposal is ready, add your file to this repo by making a PR. The number of your PR will become the number of your 3IP."